keyboards/xd75/keymaps/daniel/config.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CONFIG_USER_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

Reply:
This file I think can be deleted, as with the other keymap `config.h` files, for the same reasons as before.